src/wp-includes/block-supports/background.php

```diff
@@ -58,9 +58,9 @@ function wp_render_background_support( $block_content, $block ) {
 		return $block_content;
 	}
 
-	$background_image_source = _wp_array_get( $block_attributes, array( 'style', 'background', 'backgroundImage', 'source' ), null );
-	$background_image_url    = _wp_array_get( $block_attributes, array( 'style', 'background', 'backgroundImage', 'url' ), null );
-	$background_size         = _wp_array_get( $block_attributes, array( 'style', 'background', 'backgroundSize' ), 'cover' );
+	$background_image_source = $block_attributes['style']['background']['backgroundImage']['source'] ?? null;
```

Comment:
Thank you for the feedback @hellofromtonya!

That ticket is about converting all existing `isset( $var ) ? $var : 'default'` checks in Core to use `??` where appropriate. It has nothing to do with this PR - which removes unnecessary calls to `_wp_array_get`. It should not be a part of that ticket... the fact that both PRs use `??` is conincidental. We could convert this PR to use `isset()` checks and it would be just as valuable and valid as it is now.

The fact that we don't _yet_ use `??` anywhere in Core does not mean that it's not allowed, or that it should not be used. When WordPress had to support PHP 5.6 we **could not** use it because we had to be compatible with that PHP version. However, "_could not use_" is not the same as "_WordPress does not allow_".

The `??`operator is a part of the PHP language - just like `==` or `&&`. The only difference is the supported PHP versions, something which is no longer blocking us.